website/content/en/docs/ansible/information-flow-ansible-operator.md

```diff
@@ -1,6 +1,10 @@
-# Information Flow for Operator SDK Ansible-based Operators
+---
+title: Information Flow for Operator SDK Ansible-based Operators
+linkTitle: Information Flow
+weight: 10
+---
 
-![Ansible Operator Flow](ao-flow.png)
+![Ansible Operator Flow](/ao-flow.png)
```

Comment:
This works because I've also moved ao-flow into the static dir.

website/content/en/docs/ansible/ao-flow.png → website/static/ao-flow.png